Application/src/Main.java

```diff
@@ -30,12 +30,18 @@ public void start(Stage primaryStage) {
             public void handle(ActionEvent event) {
                 Collection<String> result = Import.file(primaryStage);
                 if (result == null) return;
+
+                
+
 				// Check if the JSON file contains a supported diagram type.
                 System.out.println(DiagramCheck.ContainsDiagram(result));
-                // TODO remove print line and parse result (user story 5)
-                System.out.println(result);
+                // Parse the element if it contains a sequence_diagram
+				if (DiagramCheck.ContainsDiagram(result).equals("sequence_diagram")){
```

Comment:
Would prefere a case here instead so its easier to extend later if needed but no need to change for now

Reply:
Or ignore this